buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateIntellijStableVersions.kt

```diff
@@ -0,0 +1,66 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.getMajorPart
+import com.virtuslab.gitmachete.buildsrc.IntellijVersionHelper.versionIsNewerThan
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import org.jsoup.Jsoup
+
+open class UpdateIntellijStableVersions : DefaultTask() {
+  @Internal
+  val intellijReleasesUrl: String = "https://www.jetbrains.com/intellij-repository/releases/"
+
+  private fun findMatchingVersionNewerThan(regex: Regex, thresholdVersion: String): String? {
```

Comment:
Pls make the naming consistent with `UpdateEapBuildNumber` (actually, I like `find` better than `check`), and consider DRYing up shared logic b/w two classes